templates/miq-template.yaml

```diff
@@ -442,7 +585,7 @@ parameters:
     displayName: "Application Min CPU Requested"
     required: true
     description: "Minimum amount of CPU time the Application container will need (expressed in millicores)."
-    value: "1000m"
+    value: "900m"
```

Comment:
Or was this decreased because we won't be using as much CPU because we removed Apache?

Does it make sense to decrease the minimum memory requested also? Maybe we need some real numbers on the idle memory usage with and without apache.